source/CardGames.py

```diff
@@ -280,12 +280,38 @@ def winnnerCheck(self, house, player):
       return 1 if houseHand <= 21 else 2
 #####################################################################################################################
 """Initializations"""
-deck = Deck()
+
+player_name = input("Username: ")
```

Comment:
Maybe let the user know they can just type anything to continue. ex. player_name = input("Username: (type anything)   ")

Reply:
noted